modules/ROOT/pages/index.adoc

```diff
@@ -9,60 +9,45 @@ With Anypoint API Governance you can:
 
 * *Improve your organization's API quality:* Identify conformance issues in governed APIs and take steps to resolve them.
 * *Share and enforce governance best practices:* Customize and publish governance rulesets to share and enforce organization-specific best practices with your developers.
-* *Apply consistent rules from design time to deployment:* Apply governance rulesets from design time to deployment using centralized governance.
+* *Apply consistent rules from design time to deployment:* Use governance rulesets to apply centralized governance to multiple aspects of your APIs, from specifications at design time to instances at deployment.
 * *Enforce governance within your DevOps organization:* Automate API governance using CLI commands in your CI/CD pipeline.
 
+== Anypoint API Governance Overview Video 
+
+Watch the Anypoint API Governance Product Spotlight video to see a quick overview of Anypoint API Governance.
+
+++++
+include::_partials/api-gov-gs-video.html[]
+++++
+
 == Governance Console
 
-In the API Governance console, add <<gov-rulesets,_governance rulesets_>> to <<gov-profiles,_governance profiles_>> to apply the governance rulesets to multiple APIs across your organization. The API Governance console then provides you with an overview of conformance for all validated APIs.
-Monitor your APIs' conformance and notify developers to help improve the conformance.
+In the API Governance console, governance administrators can add <<gov-rulesets,governance rulesets>> to <<gov-profiles,governance profiles>> to apply the governance rulesets to multiple APIs across your organization. The API Governance console then provides you with an overview of conformance for all validated APIs.
```

Comment:
Should "your" be "their" in this case? Or maybe something like "as a governance administrator, you can add..."

Reply:
Yes, this does switch voice. Again, it's picking between various evils. More wordy, 2nd voice, but having a role indicated.

Reply:
You could change the "your" to "the" to avoid the voice switch. And remove the "you with" in the second sentence.

Reply:
I reworded slightly. Hopefully it helps. The new role makes it a little more awkward to talk about.